ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -227,15 +243,30 @@ struct FrameworkBuilder {
       """)
 
       // Use the Xcode-generated path to return the path to the compiled library.
-      let libPath = buildDir.appendingPathComponents(["Release-\(platform.rawValue)",
+      let realFramework = realFrameworkName(framework)
+      let libPath = buildDir.appendingPathComponents(["Release-\(platformFolder)",
                                                       framework,
-                                                      "lib\(framework).a"])
+                                                      "\(realFramework).framework"])
+
+      print("buildThin returns \(libPath)")
       return libPath
     }
   }
 
+    // Cries in Google. Why is this not the same?
```

Comment:
This is likely because of CocoaPods `module_name` overrides like https://github.com/protocolbuffers/protobuf/blob/master/Protobuf.podspec#L15